scripts/example-pipelines-test.sh

```diff
@@ -1,3 +1,102 @@
 #!/bin/bash
 
-echo "Temporary test file"
+cp -L $KUBECONFIG /tmp/kubeconfig && export KUBECONFIG=/tmp/kubeconfig
+export IMG=${CI_OPERATOR_IMG}
+
+winImageDownloadURL="http://http-server/disk.img"
+
+# Deploy resources
+echo "Deploying resources"
+./scripts/deploy-resources.sh
+
+# SECRET
+accessKeyId="/tmp/secrets/accessKeyId"
+secretKey="/tmp/secrets/secretKey"
+
+if test -f "$accessKeyId" && test -f "$secretKey"; then
+  id=$(cat $accessKeyId | tr -d '\n' | base64)
+  token=$(cat $secretKey | tr -d '\n' | base64 | tr -d ' \n')
+
+  oc apply -n kubevirt -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tekton-operator-container-disk-puller
+  namespace: kubevirt
+type: Opaque
+data:
+  accessKeyId: "${id}"
+  secretKey: "${token}"
+EOF
+fi
+
+function wait_until_exists() {
+  timeout 10m bash <<- EOF
+  until oc get $1; do
+    sleep 5
+  done
+EOF
+}
+
+function wait_for_pipelinerun() {
+  oc wait -n kubevirt --for=condition=Succeeded=True pipelinerun -l pipelinerun=$1-run --timeout=60m &
+  success_pid=$!
+
+  oc wait -n kubevirt --for=condition=Succeeded=False pipelinerun -l pipelinerun=$1-run --timeout=60m && exit 1 &
+  failure_pid=$!
+
+  wait -n $success_pid $failure_pid
+
+  if (( $? == 0 )); then
+    echo "Pipelinerun $1 succeeded"
+  else
+    echo "Pipelinerun $1 failed"
+    exit 1
+  fi
+}
+
+# Disable smart cloning
+oc patch cdi cdi --type merge -p '{"spec":{"cloneStrategyOverride":"copy"}}'
```

Comment:
Why do we need to disable smart cloning?

Reply:
according to cdi team, it looks like, the smart cloning has some issue in azure clusters. It is ok to disable the smart cloning, only the cloning of pvs will be slower.

Reply:
caused by https://issues.redhat.com/browse/CNV-21844

Reply:
Can you add the issue to the comment in the code?

/hold